web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -348,21 +347,22 @@ DataFrames. Therefore, we decided to keep the `inplace` keyword for this small s
 ### Standardize on the `copy` keyword instead of `inplace`
 
 It may seem more natural to standardize on the `copy` keyword instead of the `inplace` keyword, since the `copy`
-keyword already returns a new object instead of None (enabling method chaining) when it is set to `True`.
+keyword already returns a new object instead of None (enabling method chaining) and avoids a coopy when it is set to `False`.
```

Comment:
```suggestion
keyword already returns a new object instead of None (enabling method chaining) and avoids a copy when it is set to `False`.
```
typo :)